swank/rpc.lisp

```diff
@@ -8,7 +8,7 @@
 ;;; are disclaimed.
 ;;;
 
-(in-package swank/rpc)
+(in-package :swank/rpc)
```

Comment:
Helmut actually meant to write it like that. If we change it here, we should then change it in a bunch of other places.

Reply:
> As it stands, it's equally as awful to make a change for code that doesn't exist.

Well, this is a single place that doesn't accept a `string` but has more narrow restrictions.
I think ensuring that type within the same library isn't wrong.


> Helmut actually meant to write it like that. If we change it here, we should then change it in a bunch of other places.

I just noticed that even within swank it's not one-or-another. Should I amend my PR to change all over to keywords, as it seems to be custom (data are my QL libraries)?